src/mudlet-lua/lua/geyser/GeyserAdjustableContainer.lua

```diff
@@ -810,15 +812,14 @@ function Adjustable.Container:new(cons,container)
     if me.locked then
         me:lockContainer()
     end
-
     me.adjLabelstyle = me.adjLabelstyle..[[ qproperty-alignment: 'AlignLeft | AlignTop';]]
-    me.lockLabel.txt = me.lockLabel.txt or [[<font size="5" face="Noto Emoji">🔒</font> Lock/Unlock]]
-    me.minLabel.txt = me.minLabel.txt or [[<font size="5" face="Noto Emoji">🗕</font> Min/Restore]]
-    me.saveLabel.txt = me.saveLabel.txt or [[<font size="5" face="Noto Emoji">💾</font> Save]]
-    me.loadLabel.txt = me.loadLabel.txt or [[<font size="5" face="Noto Emoji">📁</font> Load]]
-    me.attLabel.txt  = me.attLabel.txt or [[<font size="5" face="Noto Emoji">⚓</font> Attach to:]]
-    me.lockStylesLabel.txt = me.lockStylesLabel.txt or [[<font size="5" face="Noto Emoji">🖌</font> Lockstyle:]]
-    me.customItemsLabel.txt = me.customItemsLabel.txt or [[<font size="5" face="Noto Emoji">🖇</font> Custom:]]
+    me.lockLabel.txt = me.lockLabel.txt or [[<font size="5" face="Noto Emoji">🔒</font>]] .. self.Locale.lock.message
```

Comment:
This hard encoded use of Noto Emoji is only valid for Linux builds - other OSes cannot (Windows 10)/do not (MacOs) use that font...

Reply:
That's OK - it tested and emoji shows up fine on all OSs. So it's harmless for others.

Reply:
> This hard encoded use of Noto Emoji is only valid for Linux builds - other OSes cannot (Windows 10)/do not (MacOs) use that font...

I'm aware of that.
This is just a workaround to always have emojis with colour in Linux.
It happens that they lost their colour if the adjustable container was created in a userwindow or a new Style-sheet was set.